src/main/java/frc/robot/Constants.java

```diff
@@ -74,6 +75,62 @@ public static class EndEffectorConstants {
     public static final Angle END_EFFECTOR_MIN_RPM = Rotations.of(50);
   }
 
+  public static final class ElevatorConstants {
+    public static final int ELEVATOR_MOTOR = 22;
+    public static final Current ELEVATOR_CURRENT_LIMIT = Amp.of(40);
+    public static final double ELEVATOR_P_VALUE = 0.15;
+    public static final double ELEVATOR_D_VALUE = 0.0;
+    public static final int ELEVATOR_INTAKE_SETPOINT = 0;
+    public static final int ELEVATOR_IDLE_SETPOINT = 0;
+    public static final int ELEVATOR_L1_SETPOINT = 0;
+    public static final int ELEVATOR_L2_SETPOINT = 15;
+    public static final int ELEVATOR_L3_SETPOINT = 34;
+    public static final int ELEVATOR_L4_SETPOINT = 0;
+    public static final int ELEVATOR_THRESHOLD = 1;
+    public static final int ELEVATOR_ZERO_ENCODER = 0;
+  }
+
+  public static final class ArmConstants {
+    public static final int ARM_MOTOR = 21;
+    public static final Current ARM_CURRENT_LIMIT = Amp.of(40);
+    public static final double ARM_P_VALUE = 0.00005;
+    public static final double ARM_D_VALUE = 0.0;
+    public static final int ARM_INTAKE_SETPOINT = 0;
+    public static final int ARM_IDLE_SETPOINT = 0;
+    public static final int ARM_L1_SETPOINT = 0;
+    public static final int ARM_L2_SETPOINT = 5;
+    public static final int ARM_L3_SETPOINT = 9;
+    public static final int ARM_L4_SETPOINT = 0;
+    public static final int ARM_ZERO_ENCODER = 0;
+  }
+
+  public static final class SimulationRobotConstants {
+    public static final double PIXELSPERMETER = 20;
```

Comment:
Are there units for this?

Reply:
Not to my knowledge

Reply:
@JetBluefan1 then do 
```suggestion
    public static final double PIXELS_PER_METER = 20;
```